### DISCUSSION TITLE
Stuck on deployment

### DISCUSSION THREAD

**Discussion by bachelor-emgi** [community/community#36974] [Actions] [answered]
Hello, I have been recently deleting few of my files in repository (yes manually) and during that github pages was updating webpage with each file change. And now when I want to have published latest version, I just get this error at deployment: 
`Current status: deployment_queued`

It is running just in a loop printing same error... I have no idea how to fix it since it is my very first experience with github actions when it comes to errors.
![Snímka obrazovky (84)](https://user-images.githubusercontent.com/26904790/197352518-8976ce80-beb3-416c-a3a1-2500ef33aa7f.png)

Repository link I am running it on: https://github.com/marek-guran/marek-guran.github.io

**Comment by bachelor-emgi** [comment, marked as answer]
Looks like it has fixed itself after 1 hour of me trying to make it work. Probably there was some kind of cool down of me halting system.

**Comment by ossanna16** [comment]
Hi there @marek-guran and welcome to our community 😄. Thanks so much for asking a great question!
